lib/icalendar/event.ex

```diff
@@ -6,6 +6,7 @@ defmodule ICalendar.Event do
   defstruct summary: nil,
             dtstart: nil,
             dtend: nil,
+            dtstamp: nil,
```

Comment:
Does it crash when this value is nil? Is there a default value we can use to avoid a breaking API change?

Reply:
It's a required attribute for the iCalendar spec, not this lib. :)

My idea would be to add a `validate/1` function at some point that works on an ICalendar struct to see if it's correct, then enforce this check on a major version.

Reply:
Aye, but I'd like to avoid unclear breaking changes in the library API. I would like to a detailed warning and fall back to a default value if the programmer has not supplied one.

Reply:
Having said that, I've just noticed we're not at version 1 yet. Can make this crash with a useful error message and then cut a new major version.

Are there any other breaking changes we may want to include?

Reply:
What do want to do here? :)

Reply:
@lpil Sorry about the *very* long answer. I added a default value (the current UTC date) for DTSTAMP when it's being serialized.

Reply:
Great, thank you. Would you mind updating the CHANGELOG file too?

Reply:
Added to an Unrelased section in the CHANGELOG

Reply:
What prevents this PR from being merged?